docs/osf-trivia-list/dasharo.md

```diff
@@ -126,3 +126,26 @@ applications.
 [cb-distro]: https://doc.coreboot.org/distributions.html
 [intel-quark]: https://mail.coreboot.org/hyperkitty/list/coreboot@coreboot.org/thread/YRJQIPVK5WHACT64TH42CLGD4TXG3XTS/#PZUIFZZHRK7M3NLBNLI6VUBD4O52245B
 [vpub]: https://vpub.dasharo.com/
+
+## What is Dasharo binary blob policy?
+
+Modern x86 platforms' firmware requires closed source blobs to be integrated
+into the image to properly initialize the silicon. The ecosystem is shifting
+towards designs and technologies with a lot of small microcontrollers and
+intellectual property (IP) blocks specialized in a very thin range of tasks.
+Those microcontrollers and IP blocks typically require firmware blobs as well.
+Some of the blobs are clearly visible, some may be obfuscated and hidden inside
+the silicon or other firmware blobs (e.g. Intel Management Engine region
+contains multiple other blobs besides the ME firmware -
+[more about Intel ME blob](me.md)).
+
+So Dasharo's binary blob policy is as follows:
+
+> Integrate only the necessary amount of blobs required for proper platform
+> operation and minimize the amount of blobs that are optional whenever
+> possible by providing open equivalent implementations or removing them if
+> there is no functional impact on the platform operation.
+
+Dasharo also works without blobs on platforms that allow that. For example,
+ASUS KGPE-D16 can run without any blobs (officially there is no PSP on that
+hardware, and Opteron 6200 series CPUs can run without microcode patches).
```

Comment:
We should mention Talos II probably?

Reply:
Talos II has some blobs IIRC. @krystian-hebel am I correct?

Reply:
Everything can be build from open-source code, but we don't bother with building everything for coreboot and reuse what Hostboot has built. We treat it as a blob, but with few more hours we could build everything from source.

Reply:
I recall some hcode blobs or other things that didn't have source code. Obviously not asking for things that have source code but we do not build it.

Reply:
There are some blobs for memory controllers on scale-up variants of POWER9, but those are not used by Talos II because it uses scale-out processors with directly attached DDR4 memory.